discover/reference/articles/knowledge-base-configuration.markdown

```diff
@@ -0,0 +1,683 @@
+			</td>
+			<td>
+					emailKBSuggestionResolvedBody<br>
+					emailKBSuggestionResolvedEnabled<br>
+					emailKBSuggestionResolvedSubject<br>
+			</td>
+		</tr>
+		<tr>
+			<td>
+				        admin.email.kb.article.updated.body<br>
+					admin.email.kb.article.updated.enabled<br>
+					admin.email.kb.article.updated.resolved.subject<br>
+			</td>
+			<td>
+					emailKBSuggestionUpdatedBody<br>
+					emailKBSuggestionUpdatedEnabled<br>
+					emailKBSuggestionUpdatedSubject<br>
+			</td>
+		</tr>
+		<tr>
+			<td>
+				        admin.kb.article.increment.priority.enabled
+			</td>
+			<td>
+					articleIncrementPriorityEnabled
+			</td>
+		</tr>
+		<tr>
+			<td>
+				        markdown.importer.article.extensions
+			</td>
+			<td>
+					markdownImporterArticleExtensions
+			</td>
+		</tr>
+		<tr>
+			<td>
+				        markdown.importer.article.intro
+			</td>
+			<td>
+					markdownImporterArticleIntro
+			</td>
+		</tr>
+		<tr>
+			<td>
+				        markdown.importer.image.file.extensions
+			</td>
+			<td>
+					markdownImporterImageFileExtensions
+			</td>
+		</tr>
+		<tr>
+			<td>
+				        markdown.importer.image.folder
+			</td>
+			<td>
+					markdownImporterImageFolder
+			</td>
+		</tr>
+		<tr>
+			<td>
+				        knowledge.base.source.url.enabled
+			</td>
+			<td>
+					sourceURLEnabled
+			</td>
+		</tr>
+		<tr>
+			<td>
+				        knowledge.base.source.url.edit.message.key
+			</td>
+			<td>
+					sourceURLEditMessageKey
+			</td>
+		</tr>		
+	</tbody>
+</table>
+</div>
+</div>
+</div>
+</div>
+</div>
+
+The properties below are for the Knowledge Base Section configuration file:
+
+<div class="system-settings-table">
+<div class="max-960">
+<div class="aui-helper-clearfix section-1" >
+<div class="aui-w100 block-1 content-column" >
+<div class="content-column-content">
+<table>
+        <caption>Knowledge Base Section</caption>
+	<thead>
+		<td class="table-header left-header">
+			portlet property
+		</td>
+		<td class="table-header second">
+			System Setting
+		</td>
+	</thead>
+	<tbody>
+		<tr>
+			<td>
+				        admin.kb.article.sections
+			</td>
+			<td>
+					adminKBArticleSections
+			</td>
+		</tr>
+		<tr>
+			<td>
+				        admin.kb.article.sections.default
+			</td>
+			<td>
+					adminKBArticleSectionsDefault
+			</td>
+		</tr>
+	</tbody>
+</table>
+</div>
+</div>
+</div>
+</div>
+</div>
+
++$$$
+
+**Note:** The `knowledge.base.ratings.number.of.stars` property has been removed 
+since Knowledge Base now has a Ratings Configuration Framework. You can change 
+the ratings type in the Instance Settings configuration menu under the *Social* 
+tab. You can also configure the ratings type for an individual site under the 
+*Social* tab of the Site Settings configuration menu.
+
+The `knowledge.base.social.bookmarks.display.style` property can now be found
+under the *Social bookmarks display style* setting of each Knowledge Base 
+app's configuration menu in System Settings.
+
+$$$
```

Comment:
This information is necessary for administrators who were using these properties. For this reason, it would be best to pull them out of the sidebar and even draw attention to them with a heading that indicates these properties have been removed. I suggest listing each property in a bullet or in bold, and describing what's replaced them.